docs/introduction/plan.md

```diff
@@ -0,0 +1,36 @@
+# Operational Planning
+
+An operational plan is a schedule of related operational processes, that constitute a body of scheduled work with defined deliverable(s).  A plan normally contains one or more process resource flows, one for each deliverable.
+
+A plan can cover more than one scope, if the different scopes are tightly coordinated with pre-agreed rules, for example sub-organizations of a main organization, or an ongoing supply chain.  If not, or if the agents prefer, then requirements from another scope should become deliverables for that scope's plan.  Different batch sizes could trigger a new plan for inputs to the main deliverable too.  But all of this does not affect the vocabulary or model.  Plans can relate to each other through resource flows just like Processes.
+
+Plans are used for understanding and coordinating what needs to happen for specific outputs. The size and complexity of a Plan is up to the people who are planning and coordinating the work.
+
+A Plan can be generated from a Recipe, or created without one.
+
+Some examples:
+
+* A communications group publishes posts for the larger group.  The communications group needs some of their posts translated into various languages.
```

Comment:
Suggestion:
* A communications group creates articles for a larger group.  The communications group needs some of their articles to be translated into various languages. Both the creation of an article and its translation could be part of the same plan.

Reply:
changed